proxy/proxies/nginx/nginx.go

```diff
@@ -602,24 +632,24 @@ func (f *nginx) GenerateDeployment(lb *netv1alpha1.LoadBalancer) *extensions.Dep
 								"--healthz-port=" + "10254",
 								"--health-check-path=" + "/healthz",
 							},
-							ReadinessProbe: &v1.Probe{
-								Handler: v1.Handler{
-									HTTPGet: &v1.HTTPGetAction{
-										Path:   "/ingress-controller-healthz",
-										Port:   intstr.FromInt(10254),
-										Scheme: v1.URISchemeHTTP,
-									},
-								},
-							},
-							LivenessProbe: &v1.Probe{
-								Handler: v1.Handler{
-									HTTPGet: &v1.HTTPGetAction{
-										Path:   "/ingress-controller-healthz",
-										Port:   intstr.FromInt(10254),
-										Scheme: v1.URISchemeHTTP,
-									},
-								},
-							},
+							// ReadinessProbe: &v1.Probe{
```

Comment:
I have no idea why health check can not work now, and I don't have the time to track down the problem. So I remove it temporarily